unlock-app/src/components/interface/checkout/main/checkoutMachine.ts

```diff
@@ -154,7 +154,7 @@ type Payment =
     }
   | {
       method: 'swap_and_purchase'
-      route: any
+      route?: object
```

Comment:
```suggestion
      route?: any
```

I will keep any here or find a route type from uniswap sdk.